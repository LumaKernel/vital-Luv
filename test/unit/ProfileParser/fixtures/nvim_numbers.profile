SCRIPT  /home/luma/vim-make-plugin/vital-Luv/dev/ProfileParser/test_numbers.vim
Sourced 1 time
Total time:   0.000066
 Self time:   0.000066

count  total (s)   self (s)
                            
    1              0.000005 123
    1              0.000002 123  456  789
                            

