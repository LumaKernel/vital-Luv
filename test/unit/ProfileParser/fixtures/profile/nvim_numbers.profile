SCRIPT  /test/test_numbers.vim
Sourced 1 time
Total time:   0.000059
 Self time:   0.000059

count  total (s)   self (s)
                            
    1              0.000006 123
    1              0.000002 123  456  789
                            

