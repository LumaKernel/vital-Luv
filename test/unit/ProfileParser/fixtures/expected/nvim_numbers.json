[{"path":"/test/test_numbers.vim","total_time":5.9e-5,"self_time":5.9e-5,"type":"script","lines":[{"content":""},{"self_time":6.0e-6,"count":1,"content":"123"},{"self_time":2.0e-6,"count":1,"content":"123  456  789"},{"content":""}],"count":1}]
